js/apps/admin-ui/cypress/e2e/users_test.spec.ts

```diff
@@ -143,6 +145,14 @@ describe("User creation", () => {
     listingPage.searchItem(itemId).itemExist(itemId);
   });
 
+  it("Select Unmanaged attributes", () => {
+    sidebarPage.goToRealmSettings();
+    cy.wait(1000);
```

Comment:
We prefer not to do arbitrary waits as it can slow down the tests unnecessarily. Perhaps it would be possible to replace this with a page load wait?
```suggestion
    sidebarPage.waitForPageLoad();
```